lib/generators/react/install_generator.rb

```diff
@@ -110,6 +110,12 @@ def setup_react_webpacker
           create_file(manifest, WEBPACKER_SETUP_UJS)
         end
       end
+
+      private
+
+      def webpack_configuration
+        Webpacker.respond_to?(:config) ? Webpacker.config : webpack_configuration
```

Comment:
typo. should be
```
Webpacker.respond_to?(:config) ? Webpacker.config : Webpacker::Configuration
```

Reply:
Aaw whoops. Wondered why my test didn't pass XD Thanks

Reply:
I've still got issues around `Webpacker::Manifest.load`, seems to be how I detect `respond_to?` may be wrong for that one.